### PR TITLE
[0] Store Lookup, Clerk Login and Customer Lookup styling

### PR DESCRIPTION
**Pull request by tommypoa**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
- Styling for `StoreLookupScreen` (new), `ClerkLoginScreen` and `CustomerLookupScreen`
   - New back button 
- Removed picker, store defaulted to "A & S Grocery" for now (to be resolved by PR #16)
- `checkin.js` for styles

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Online sources

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"
- Nested headers: https://github.com/react-navigation/react-navigation/issues/614

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
- Alignment to Figma
- Catching bugs by going through entire log-in flow
- Code quality

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"
- Vertical alignment for each of the pages are not uniformed now
## Tests Performed, Edge Cases

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"
- Ran through the entire flow without any errors!
### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"
`StoreLookupScreen`
image
`ClerkLoginScreen`
image
`CustomerLookupScreen`
image

CC: @wangannie @thumn 

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: components/BackButton.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { View } from 'react-native';
+import { LineItem, LineItemRow } from '../styled/checkout.js';
```
[the remark this answers is not in the document]
Comment (anniero98) [reply components/BackButton.js]: I fixed this in the second PR lol (`organizeImports` by VS Code ,, pls

File: screens/ClerkLoginScreen.js [hunk 2]
```diff
@@ -80,33 +77,40 @@ export default class ClerkLoginScreen extends React.Component {
...
-            keyboardType="number-pad"
-            maxLength={4}
-            secureTextEntry
-            onChangeText={text => this.setState({ password: text })}
-            value={this.state.password}
-          />
-          <SubmitButton color="#008550" title="Log In" onPress={() => this.handleSubmit()} />
-          {this.state.errorMsg ? <Text>{this.state.errorMsg}</Text> : null}
+        <CheckInContainer>
+          <BackButton navigation={this.props.navigation} />
```
[the remark this answers is not in the document]
Comment (anniero98) [reply screens/ClerkLoginScreen.js]: I fixed this in the next PR also

File: components/BaseComponents.js [hunk 3]
```diff
@@ -59,4 +59,5 @@ export const FilledButtonContainer = styled(ButtonContainer)`
   width: ${props => props.width || '309px'};
   height: ${props => props.height || '57px'}
   background: ${props => (props.color ? props.color : Colors.primaryGreen)};
+  border-radius: 20px;
```
[the remark this answers is not in the document]
Comment (anniero98) [reply components/BaseComponents.js]: I made a styled component for this in #14 called `RoundedButtonContainer` 🤷‍♀
